README.md

```diff
@@ -201,3 +207,12 @@ resume normal operation.
 In any case, `clboss-ignore-onchain` is temporary and even
 if you forget to issue `clboss-notice-onchain` CLBOSS will
 resume managing onchain funds at some point.
+
+
+### Configuration
```

Comment:
The `clboss-min-onchain` pull request already includes a description of `clboss-min-onchain` in the `README.md`, but I think separating the configuration options might make more sense (the `clboss-min-onchain` PR adds it next to the various commands).  That probably makes this PR conflict with the already-merged `clboss-min-onchain` PR, so maybe you can copy some of the text there to here?